src/tests/unit/jobs-dao.test.js

```diff
@@ -80,4 +81,137 @@ describe('Test jobs-dao', () => {
+      dbReturn: { rows: [{ count: 1 }], outBinds: { result: null } },
+      expected: null,
+      postBody: { studentEmployeeInd: false, changeReason: { code: 'dummy' } },
+      expectedCalledType: jobStubType.graduateJob,
+    },
+    {
+      message: 'handleJob should call studentJob',
+      dbReturn: { rows: [{ count: 1 }], outBinds: { result: null } },
+      expected: null,
+      postBody: { studentEmployeeInd: true, positionNumber: 'C50236', changeReason: { code: 'dummy' } },
+      expectedCalledType: jobStubType.studentJob,
+    },
+    {
+      message: 'handleJob should call terminateJob for student',
+      dbReturn: { rows: [{ count: 1 }], outBinds: { result: null } },
+      expected: null,
+      postBody: { studentEmployeeInd: true, positionNumber: 'C50236', changeReason: { code: 'TERMJ' } },
+      expectedCalledType: jobStubType.terminateJob,
+    },
+    {
+      message: 'handleJob should call terminateJob for graduate',
+      dbReturn: { rows: [{ count: 1 }], outBinds: { result: null } },
+      expected: null,
+      postBody: { studentEmployeeInd: false, positionNumber: 'C50236', changeReason: { code: 'TERMJ' } },
+      expectedCalledType: jobStubType.terminateJob,
+    },
+    {
+      message: 'handleJob should call updateLaborChangeJob for graduate',
+      dbReturn: { rows: [{ count: 1 }], outBinds: { result: null } },
+      expected: null,
+      postBody: { studentEmployeeInd: false, positionNumber: 'C50236', changeReason: { code: 'NONE' } },
+      expectedCalledType: jobStubType.updateLaborChangeJob,
+    },
+    {
+      message: 'handleJob should call updateLaborChangeJob for student',
+      dbReturn: { rows: [{ count: 1 }], outBinds: { result: null } },
+      expected: null,
+      postBody: { studentEmployeeInd: true, positionNumber: 'C50236', changeReason: { code: 'NONE' } },
+      expectedCalledType: jobStubType.updateLaborChangeJob,
+    },
+  ];
+  _.forEach(handleJobTestCases, ({
+    message,
+    dbReturn,
+    expected,
+    postBody,
+    expectedCalledType,
+  }) => {
+    it(message, async () => {
+      const stubGraduateJob = sinon.stub(contrib, 'graduateJob');
+      const stubStudentJob = sinon.stub(contrib, 'studentJob');
+      const stubTerminateJob = sinon.stub(contrib, 'terminateJob');
+      const stubUpdateLaborChangeJob = sinon.stub(contrib, 'updateLaborChangeJob');
+      const daoProxy = createDaoProxy(dbReturn, dbReturn);
+
+      const result = daoProxy.handleJob(fakeOsuId, postBody);
+      await result.should.eventually.be.fulfilled.and.deep.equal(expected);
+
+      assertStubCalled(expectedCalledType === jobStubType.graduateJob, stubGraduateJob);
+      assertStubCalled(expectedCalledType === jobStubType.studentJob, stubStudentJob);
+      assertStubCalled(expectedCalledType === jobStubType.terminateJob, stubTerminateJob);
+      assertStubCalled(expectedCalledType === jobStubType.updateLaborChangeJob,
+        stubUpdateLaborChangeJob);
```

Comment:
Indentation looks weird here. Let's do this instead:

```js
assertStubCalled(
  expectedCalledType === jobStubType.updateLaborChangeJob, stubUpdateLaborChangeJob
);
```

Reply:
OK